src/jsutils/AbortController.ts

```diff
@@ -0,0 +1,38 @@
+export interface IAbortSignal {
+  aborted: boolean;
+  addEventListener: (type: string, handler: () => void) => void;
+  removeEventListener: (type: string, handler: () => void) => void;
+}
+
+export interface IAbortController {
+  signal: IAbortSignal;
+  abort: (reason?: any) => void;
+}
+
+// We need to polyfill abort controller in case of Node@14 which doesn't have it
+/* c8 ignore start */
+export const AbortController: new () => IAbortController =
+  // eslint-disable-next-line no-undef
+  global.AbortController ||
+  // This is a dummy implementation that doesn't actually abort anything
+  class DummyAbortController implements IAbortController {
+    private _signal: IAbortSignal = {
+      aborted: false,
+      addEventListener: () => null,
```

Comment:
Are there any plans on dropping support for node@14? Would be cool to track it down somewhere so that people know when it's going to happen.

Reply:
I don't know if graphql-js has any particular plans, but Node v14 does go out of LTS in April. (I'm not sure what non-Node/browser platforms graphql-js has to work on though?)

It might be reasonable to just only do the "cancel stuff when execution is over" if `global.AbortController` exists. It's a bit more annoying to do this partial support if we add an AbortSignal to GraphQLRequestInfo since it would be nice if the TS type of the field is non-optional — but we could have that field be optional for now (with a "if you know you're in an AbortController-in-global environment then you can `!` it") and non-optional once Node 14 support is dropped.